redfish-core/include/license_messages.hpp

```diff
@@ -0,0 +1,84 @@
+
+#include "http_response.hpp"
+
+#include <error_messages.hpp>
+#include <nlohmann/json.hpp>
+
+namespace redfish
+{
+
+namespace messages
+{
+
+inline nlohmann::json licenseInstalled(const std::string& arg1)
+{
+    return nlohmann::json{{"@odata.type", "#Message.v1_0_0.Message"},
+                          {"MessageId", "License.1.0.0.LicenseInstalled"},
+                          {"Message", "The license has been installed " + arg1},
+                          {"MessageArgs", {arg1}},
+                          {"Severity", "OK"},
+                          {"Resolution", "None."}};
+}
+
+void licenseInstalled(crow::Response& res, const std::string& licenseString)
+{
+    res.result(boost::beast::http::status::ok);
+    addMessageToJson(res.jsonValue, licenseInstalled(licenseString),
+                     "LicenseInstalled");
+}
```

Comment:
Add brief description - similar to in error_message.hpp?

For all methods defined  in this file